packages/angular_devkit/build_angular/src/karma/code-coverage_spec.ts

```diff
@@ -7,8 +7,19 @@
  */
 import { Architect } from '@angular-devkit/architect';
 import { normalize, virtualFs } from '@angular-devkit/core';
+import { TestResults } from 'karma';  // tslint:disable-line
```

Comment:
This is a little weird, because `no-implicit-dependencies` tslint rule insists that `karma` (or rather @types/karma) is one of the dependencies in `package.json`. Is this a normal occurrence?

Reply:
Ideally there should be a separate lint configuration for unit test files but the current linting setup does not support multiple configuration files.